helpers/filterSeats.js

```diff
@@ -60,19 +66,58 @@ module.exports = function(dust) {
       });
     }
 
+    // get the names of the best two sections from the theatre
+    function getBestSections(bandColours) {
+      var bestSections = [];
+      // add all section names to an array
+      _.forEach(bandColours, function(bands, section) {
+        bestSections.push(section);
+      });
+      // find the best section
+      bestSections = _.last(bestSections); // just Stalls
+      return bestSections;
```

Comment:
Would you even need a function here? Looks like `_.last(bandColours);` would do all of this?